tests/unit/scheduler.c

```diff
@@ -14,22 +14,6 @@
 #include "test-common.h"
 #include "nccl_ofi_scheduler.h"
 
-int create_multiplexed(size_t size,
```

Comment:
It looks like we removed an awful lot of tests.  I'd have expected to see new tests as well.  Are we sure we're covering all the behaviors?

Reply:
I modified the tests because now, we have only one function which does both RR and multiplexing. 
I have taken care of all the cases from before which includes the check for LL128 alignment.
```
- Test_Threshold_schedule
	- Verify that the message greater than rr_threshold is multiplexed
	- Verify that the three messages with rr_threshold is round-robin three times.

- Test_Multiplexing_schedule
	- Test zero Rail
	- Test One Rail
		- size = 0
		- size = align - 1
		- size = align
		- size = align + 1
	- Test Three Rail
		- size = 4 * align - 1
		- size = 4* align
		- size = 4 * align + 1
```
But I have also ensured that we test for more msg_sizes within the 4 windows.
1. msg_size is between 0 to min_stripe_size.
2. msg_size is between min_stripes and 2*min_stripe_size.
3. msg_size is between 2*min_stripes and 3*min_stripe_size.
4. msg_size is greater than 3*min_stripes